### DISCUSSION TITLE
streamlink breaks on upgrade to Ubuntu 24

### DISCUSSION THREAD

**Discussion by setec404** [streamlink/streamlink#6296] [General]
### Description

Had a good time trying to fix streamlink after going to Ubuntu 24 LTS but eventually got it to work using pipx install streamlink --force since I couldnt uninstall it using pip the symlinks were old and --force got it to update correctly.

**Comment by bastimeyer** [comment]
1. https://streamlink.github.io/install.html#linux-and-bsd

   > | | |
   > | --- | --- |
   > | [Ubuntu](https://packages.ubuntu.com/noble/streamlink) | Not recommended.<br><br>Ubuntu inherits the streamlink package from Debian, but doesn't maintain or update it.<br><br>See [Package availability](https://streamlink.github.io/install.html#package-availability). |

2. https://streamlink.github.io/install.html#package-availability

   > Packaging is not done by the Streamlink maintainers themselves except for the [PyPI package](https://streamlink.github.io/install.html#pypi-package-and-source-code), the [Windows installers + portable builds](https://streamlink.github.io/install.html#windows-binaries), and the [Linux AppImages](https://streamlink.github.io/install.html#linux-appimages).

   > Users of glibc-based Linux distros can find up-to-date Streamlink releases via the available [AppImages](https://streamlink.github.io/install.html#linux-appimages).

3. https://streamlink.github.io/install.html#pypi-package-and-source-code
